Code/GraphMol/MolOps.cpp

```diff
@@ -317,7 +317,8 @@ namespace RDKit{
     }
 
     std::vector<ROMOL_SPTR> getMolFrags(const ROMol &mol,bool sanitizeFrags,
```

Comment:
It might be worth adding a unit test to confirm that the extra properties you are copying over here actually make it.
This would be a good check for this code, but also help ensure that things keep working properly in the future if we ever end up refactoring this.